### PR TITLE
Perf improvement to `Polymer.AppLayout.scroll`

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: helpers/helpers.html [hunk 1]
```diff
@@ -85,19 +85,20 @@
...
-
-        (function updateFrame() {
+        var updateFrame = (function updateFrame() {
           var now = Date.now();
           var elapsedTime = now - startTime;
 
           if (elapsedTime < duration) {
             scrollTo(timingFn(elapsedTime, currentScrollLeft, deltaScrollLeft, duration),
                 timingFn(elapsedTime, currentScrollTop, deltaScrollTop, duration));
-            requestAnimationFrame(updateFrame.bind(this));
```
Comment (blasten) [inline helpers/helpers.html]: this was creating a lot of garbage for GC.

File: helpers/helpers.html [hunk 2]
```diff
@@ -85,19 +85,20 @@
         var deltaScrollTop = scrollTop - currentScrollTop;
         var deltaScrollLeft = scrollLeft - currentScrollLeft;
         var duration = 300;
-
-        (function updateFrame() {
+        var updateFrame = (function updateFrame() {
```
Comment (keanulee) [inline helpers/helpers.html]: Don't need the function name in the function declaration (e.g. `function() {...`). I don't know if this actually breaks anything, but it's certainly weird to have two places that declare `updateFrame`.

Comment (blasten) [inline helpers/helpers.html]: that is for the stack trace. Otherwise, it'd be an anonymous function.

Comment (keanulee) [inline helpers/helpers.html]: I figured it was for debugging, but the variable should probably take a different name so it's less ambiguous:

```javascript
var boundUpdateFrame = (function updateFrame() { ... }).bind(this);
```

Comment (blasten) [inline helpers/helpers.html]: @keanulee I could do that, but in this case in particular making the distinction doesn't seem to provide any benefit.  wdyt? I guess arrow functions fix cases like this, but we don't use them yet.

**Comment by keanulee** [comment]
LGTM

**Pull request merged**